RecoBTag/Configuration/python/RecoBTag_EventContent_cff.py

```diff
@@ -66,6 +69,9 @@
         'keep *_pfDeepCSVJetTags_*_*',
         # DeepCMVA
         'keep *_pfDeepCMVAJetTags_*_*',
+        # DeepFlavour (new)
+        'keep *_pfDeepFlavourTagInfos_*_*',
```

Comment:
around PU35 I see that this is about 2.5% of the RECO.
I think this is too much for this purpose for this data tier.
a product next in size is particleFlow PFCandidates.